boards/common/nrf52/Makefile.include

```diff
@@ -8,7 +8,11 @@ ifeq (bmp,$(PROGRAMMER))
   PORT_DARWIN ?= $(wordlist 2, 2, $(sort $(wildcard /dev/tty.usbmodem*)))
 endif
 
-PROGRAMMER ?= jlink
+ifneq (,$(command -v jlink))
+  PROGRAMMER ?= jlink
```

Comment:
I don't think this works. The jlink command is not jlink (at least on Linux).

https://github.com/RIOT-OS/RIOT/blob/c485bd4d92a1acf81195eb47961317d58b8319bd/dist/tools/jlink/jlink.sh#L71

Reply:
Good catch!
```suggestion
  PROGRAMMER ?= JLinkExe
```

Reply:
Shouldn't it be backwards?  e.g.: `ifneq (,$(command -v JLinkExe))` and `PROGRAMMER ?= jlink`.

I would also add an additional `ifneq (,$(command -v $JLINK))` if the user specifies the `JLinkExe` executable manually without having to set `PROGRAMMER`. e.g.: `JLINK=/path/JLinkExe make flash`

Reply:
> Shouldn't it be backwards? e.g.: `ifneq (,$(command -v JLinkExe))` and `PROGRAMMER ?= jlink`.

Absolutely :) I confused the lines :/

> I would also add an additional `ifneq (,$(command -v $JLINK))` if the user specifies the `JLinkExe` executable manually without having to set `PROGRAMMER`. e.g.: `JLINK=/path/JLinkExe make flash`

Done.